ledger-tool/src/main.rs

```diff
@@ -2192,6 +2205,31 @@ fn main() {
                     }
                 }
             }
+            ("purge-older-slots", Some(arg_matches)) => {
+                let ending_slot = value_t_or_exit!(arg_matches, "ending_slot", Slot);
+                let blockstore = open_blockstore(
+                    &ledger_path,
+                    AccessType::Primary,
+                    wal_recovery_mode,
+                    &shred_storage_type,
+                    force_update_to_open,
+                );
+                match blockstore.run_purge(
+                    0, // from_slot
+                    ending_slot,
+                    PurgeType::CompactionFilter,
+                ) {
```

Comment:
The existing `purge` command can accomplish the same behavior; I believe the following are equivalent:
```
solana-ledger-tool purge --start_slot 0 --end_slot SLOT ...
solana-ledger-tool purge-older-slots --ending_slot SLOT ...
```
As such, I don't think this warrants a new subcommand unless there is some subtlety that I'm overlooking

Reply:
I didn't notice we already have the purge command.  Then I will just improve the comment saying if the starting slot is 0, then all files whose slot range is older than the ending_slot will be deleted immediately.